sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,712 @@
+
+### Install the library
+To install the Azure client libraries for Android, add them as dependencies within your
+[Gradle](#add-a-dependency-with-gradle) or
+[Maven](#add-a-dependency-with-maven) build scripts.
+
+#### Add a dependency with Gradle
+To import the library into your project using the [Gradle](https://gradle.org/) build system, follow the instructions in [Add build dependencies](https://developer.android.com/studio/build/dependencies):
+
+Add an `implementation` configuration to the `dependencies` block of your app's `build.gradle` or `build.gradle.kts` file, specifying the library's name and the version you wish to use:
+
+```gradle
+// build.gradle
+dependencies {
+    ...
+    implementation "com.azure.android:azure-communication-chat:1.0.0-beta.1"
+}
+
+// build.gradle.kts
+dependencies {
+    ...
+    implementation("com.azure.android:azure-communication-chat:1.0.0-beta.1")
+}
+```
+
+#### Add a dependency with Maven
+To import the library into your project using the [Maven](https://maven.apache.org/) build system, add it to the `dependencies` section of your app's `pom.xml` file, specifying its artifact ID and the version you wish to use:
+
+```xml
+<dependency>
+  <groupId>com.azure.android</groupId>
+  <artifactId>azure-communication-chat</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+
+## User and User Access Tokens
+
+User access tokens enable you to build client applications that directly authenticate to Azure Communication Services. Refer [here](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens) to learn how to create a user and issue a User Access Token.
+
+The id for the user created above will be necessary later to add said user as a member of a new chat thread. The initiator of the create request must be in the list of members of the chat thread.
+
+## Create the AzureCommunicationChatClient
+
+```java
+import com.azure.android.communication.chat.AzureCommunicationChatServiceAsyncClient;
+import com.azure.android.core.http.HttpHeader;
+
+final String endpoint = "https://<resource>.communication.azure.com";
+final String userAccessToken = "<user_access_token>";
+
+AzureCommunicationChatServiceAsyncClient client
+    = new AzureCommunicationChatServiceAsyncClient.Builder()
+    .endpoint(endpoint)
+    .credentialInterceptor(chain -> chain.proceed(chain.request()
+        .newBuilder()
+        .header(HttpHeader.AUTHORIZATION, userAccessToken)
+        .build());
+```
+
+# Key concepts
+
+A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages.
+
+Once you initialize an `AzureCommunicationChatClient` class, you can perform the following chat operations:
+
+## Thread Operations
```

Comment:
Let's make these H3 or H4s so that it renders better.

Reply:
updated